README.md

```diff
@@ -23,14 +23,14 @@ module.exports = {
     'filename-rules',
   ],
   rules: {
-    'filename-rules/match': [2, 'camelcase'],
+    'filename-rules/match': [2, 'camelCase'],
```

Comment:
Let's revert this example to the default style.
```suggestion
    'filename-rules/match': [2, 'camelcase'],
```

Reply:
What about the second example in readme?
```
'filename-rules/match': [2, { '.js': 'camelCase', '.ts': /^([a-z]+-)*[a-z]+(?:\..*)?$/ }]
```
I would also revert it then 😄

Reply:
Pushed changes

Reply:
Sorry for not being very clear - my idea was to keep the second as it is, so that we have an example of both in readme.

Reply:
No problem) I had the same thought. Updated